tensorboard/backend/event_processing/event_accumulator.py

```diff
@@ -136,6 +136,22 @@ class EventAccumulator(object):
 
   Histograms, audio, and images are very large, so storing all of them is not
   recommended.
+
+  Fields:
+    audios: A reservoir.Reservoir of audio summaries.
+    compressed_histograms: A reservoir.Reservoir of compressed
+        histogram summaries.
+    histograms: A reservoir.Reservoir of histogram summaries.
+    images: A reservoir.Reservoir of image summaries.
+    most_recent_step: Step of last Event proto added. This should only
+        be accessed from the thread that calls Reload.
+    most_recent_wall_time: Timestamp of last Event proto added. This
```

Comment:
Do you think that it's worth documenting the units of `timestamp` (IIRC it's `float` of seconds since epoch?)?

Do you think that it's worth documenting that these are both `-1` when no events have been read?